tests/api/admin-notes.php

```diff
@@ -314,5 +316,109 @@ public function test_get_notes_schema() {
 		$this->assertArrayHasKey( 'date_reminder_gmt', $properties );
 		$this->assertArrayHasKey( 'actions', $properties );
 		$this->assertArrayHasKey( 'is_snoozable', $properties );
+		$this->assertArrayHasKey( 'layout', $properties );
+		$this->assertArrayHasKey( 'image', $properties );
+		$this->assertArrayHasKey( 'is_deleted', $properties );
+	}
+
+	/**
+	 * Test deleting a single note.
+	 */
+	public function test_delete_single_note() {
+		wp_set_current_user( $this->user );
+
+		$response = $this->server->dispatch( new WP_REST_Request( 'GET', $this->endpoint . '/3' ) );
+		$note     = $response->get_data();
+
+		$this->assertEquals( 200, $response->get_status() );
+		$this->assertEquals( false, $note['is_deleted'] );
+
+		$response = $this->server->dispatch( new WP_REST_Request( 'DELETE', $this->endpoint . '/delete/3' ) );
+		$note     = $response->get_data();
+
+		$this->assertEquals( 200, $response->get_status() );
+		$this->assertEquals( true, $note['is_deleted'] );
+	}
+
+	/**
+	 * Test deleting a single note without permission. It should fail.
+	 */
+	public function test_delete_single_note_without_permission() {
+		$response = $this->server->dispatch( new WP_REST_Request( 'DELETE', $this->endpoint . '/delete/3' ) );
+		$note     = $response->get_data();
+
+		$this->assertEquals( 401, $response->get_status() );
+	}
+
+	/**
+	 * Test undiong a single note delete.
```

Comment:
Typo

Reply:
Cool, this was changed in the commit [be254b0](https://github.com/woocommerce/woocommerce-admin/pull/4386/commits/be254b001ef0acbb2724ef93a337f56eaeb962b3)